docs/developers/0_preparation.md

```diff
@@ -0,0 +1,22 @@
+# 💼 Preparing a development environment
+In order to compile ReVanced Manager, certain requirements must be met.
+
+## 📝 Prerequisites
+* Kotlin IDE such as [Android Studio](https://developer.android.com/studio)
+* Knowledge of [Android](https://android.com) app development, and the [Kotlin](https://kotlinlang.org/) language.
+* At least JDK 17 of any vendors
```

Comment:
`JDK 17 or above`

Reply:
```suggestion
* JDK 17 or above
```


Co-authored-by: Palm <palmpasuthorn@gmail.com>

Reply:
@PalmDevs Something feels missing, in my opinion.

"JDK 17 or above" feels inconsistent with other bullet points

I think "Any JDK 17 or above" might sound better.